Tests/MapboxMapsTests/Foundation/CredentialsManagerTests.swift

```diff
@@ -0,0 +1,29 @@
+import XCTest
+import MapboxMaps
+import MapboxCoreMaps
+
+class CredentialsManagerTests: XCTestCase {
```

Comment:
we should have some tests for the behaviors provided when you init the CredentialsManager with nil.